pkg/rosa/runtime.go

```diff
@@ -97,3 +98,17 @@ func (r *Runtime) FetchCluster() *cmv1.Cluster {
 	r.Cluster = cluster
 	return cluster
 }
+
+func (r *Runtime) DisplayClusterWarnings(clusterId string) error {
+	serviceLogs, err := r.OCMClient.ListServiceLogs(clusterId)
+	if err != nil {
+		return err
+	}
+	serviceLogs.Items().Each(func(entry *slv1.LogEntry) bool {
```

Comment:
@renan-campos I think we need to provide the user with more context here on what these warnings are, and what they need to do about them (if anything). 

For example, I'd expect _something_ like:

```
The following warnings were generated on creation of your cluster:

- Lack of Jam
 - There was a serious lack of Jam when creating your cluster. Buy some more jam (see example.com/buy-jam)

```

Reply:
Hi @robpblake the user context would be provided by the serviceLog's summary and description. 

For a little background on this PR: This warning mechanism was needed for a preflight displayed by the OCM cli. In the OCM cli, this would be the way that warning would be displayed:
```
⚠️ WARNING:
Please enable the Org Policy API for the GCP project 'REDACTED'
Without the GCP Org Policy API enabled, OCM is unable to determine whether the GCP project contains any policies that would affect the in
stallation
```

Reply:
@robpblake happy new year! With fresh eyes I had a better understanding of your comment. I added context to the warnings as requested. PTAL